buf.gen.yaml

```diff
@@ -4,6 +4,9 @@ plugins:
   - name: gapic-validator
     out: .
 
+  - name: go-aip-lint
```

Comment:
Is the linter specific for `go` or should it just be called `aip-lint` (`protoc-gen-aip-lint`).

Reply:
I'm thinking that the scope of the linter is that it should ensure that the upcoming code generator in `aip-go` will work well assuming the linter passes - so it's a linter specifically for `aip-go` - if that makes sense?

Reply:
makes sense, `protoc-gen-aip-lint` could be a bit of a broad name as well.